test-network-function/certification/suite.go

```diff
@@ -60,11 +61,21 @@ func testContainerCertificationStatus() {
 		env := configpkg.GetTestEnvironment()
 		cnfsToQuery := env.Config.CertifiedContainerInfo
 
+		if len(cnfsToQuery) == 0 {
+			ginkgo.Skip("No containers to check configured in tnf_config.yml")
+		}
+
 		ginkgo.By(fmt.Sprintf("Getting certification status. Number of containers to check: %d", len(cnfsToQuery)))
 
 		if len(cnfsToQuery) > 0 {
 			certAPIClient = api.NewHTTPClient()
+			allCnfToQueryEmpty := true
 			for _, cnf := range cnfsToQuery {
+				if cnf.Name == "" || cnf.Repository == "" {
+					tnf.ClaimFilePrintf(" cnf name = \"%s\" or repository = \"%s\" is missing, skipping this cnf to query", cnf.Name, cnf.Repository)
```

Comment:
Why do you add the extra space char at the beginning of the string to print " cnf name..."?

Reply:
fixed